Assets/Scripts/Jellies/DewInstantiate.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DewInstantiate : MonoBehaviour
+{
+
+    public GameObject Dew;
```

Comment:
Remember to avoid public fields, for encapsulation. Use [SerializeField] to expose it in the inspector.

Also, for the style guide, change Dew to _dew. It would be clearer if you name it _dewPrefab.